cli/remote_build.go

```diff
@@ -658,105 +650,52 @@ func managementLogUrl(url, org, label string) string {
 	return ""
 }
 
-func defineBranch(r *git.Repository, hintBranch string) (string, error) {
-
-	ref, err := r.Head()
-	if err != nil {
-		return "", fmt.Errorf("No Head: %v", err)
-	}
-
-	if ref.Name().IsBranch() {
-		if hintBranch != "" {
-			if hintBranch == ref.Name().Short() {
-				log.Infof("Informed branch is the one used locally")
-
-				return hintBranch, nil
-
-			} else {
-				return hintBranch, fmt.Errorf("Informed branch (%v) isn't the same as the one used locally (%v)", hintBranch, ref.Name().String())
-			}
-		} else {
-			log.Debugf("Found branch reference name is %v", ref.Name().Short())
-			return ref.Name().Short(), nil
-		}
-
-	} else {
-		return "", fmt.Errorf("No branch set?")
-	}
-}
-
-func (p *RemoteCmd) fetchProject(urls []string) (*types.Project, types.GitRemote, error) {
-	var empty types.GitRemote
+func (p *RemoteCmd) fetchProject(urls []string) (*types.Project, error) {
 	v := url.Values{}
 	fmt.Println()
 	log.Infof("URLs used to search: %s", urls)
 
 	for _, u := range urls {
-		rem := types.NewGitRemote(u)
+		rem, err := ggit.ParseURL(u)
```

Comment:
This new `err` is never exposed in the log output.

Reply:
Done.